bin/ungit

```diff
@@ -11,19 +10,10 @@ var BugTracker = require('../src/bugtracker');
 var bugtracker = new BugTracker('launcher');
 var usageStatistics = require('../src/usage-statistics');
 
-var child = new (forever.Monitor)(path.join(__dirname, '..', 'src', 'server.js'), {
-  silent: false,
-  minUptime: 2000,
-  max: config.maxNAutoRestartOnCrash,
```

Comment:
Is `maxNAutoRestartOnCrash` used anywhere else?

Reply:
Not really...  From my tests forever monitor actually never kicked in and process didn't die during errors.  Errors seems to be captured and handled so..

Reply:
Ok, so that means the config setting can be removed.